pkg/api/handlers/projects.go

```diff
@@ -474,6 +513,21 @@ func convertProject(thread *v1.Thread, parentThread *v1.Thread) types.Project {
 	return p
 }
 
+func convertProjectCapabilities(capabilities v1.ThreadCapabilities) types.ProjectCapabilities {
+	result := types.ProjectCapabilities{
+		OnSlackMessage:   capabilities.OnSlackMessage,
+		OnDiscordMessage: capabilities.OnDiscordMessage,
+		OnEmail:          capabilities.OnEmail,
+	}
+	if capabilities.OnWebhook != nil {
+		result.OnWebhook = &types.OnWebhook{}
+		result.OnWebhook.ValidationHeader = capabilities.OnWebhook.ValidationHeader
+		result.OnWebhook.Headers = capabilities.OnWebhook.Headers
+		result.OnWebhook.Secret = "********"
```

Comment:
Also, the secret is hashed, so I think it's fine to show it.
```suggestion
		result.OnWebhook = &types.OnWebhook{
			ValidationHeader: capabilities.OnWebhook.ValidationHeader
			Headers: capabilities.OnWebhook.Headers
			Secret: "********"
		}
```